app/code/core/Mage/Adminhtml/Model/Email/Template.php

```diff
@@ -80,13 +82,13 @@ public function getSystemConfigPathsWhereUsedCurrently()
         if (!$templateId) {
             return array();
         }
-        $paths = array();
 
+        $paths = array();
```

Comment:
There is another $paths at line 55. Are you sure this should be two times in a file?

Reply:
Not sure what do you mean, but at line 55 it's another method.

Reply:
You are right. I did not check the whole file.